tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -266,9 +266,11 @@ def run(self):
                     ).load_all() or []
                     volumes = []
                     for volume in all_volumes:
-                        if not str(volume.deleted).lower() == "true" or \
+                        if not str(volume.deleted).lower() == "true" and \
                             volume.current_job.get('status', '') \
```

Comment:
it should be and not or  @shtripat suggestion please

Reply:
I feel it should be as below

```
if not str(volume.deleted).lower() == 'true' or (
    volume.current_job.get('status', '') in ['', 'finished', 'failed'] and
    volume.vol_id not in [None, ''] and
    volume.name not in [None, '']
):
```

What you say?

Reply:
if a volume is deleted and other condition or true then it will proceed with deleted volume right?

Reply:
i feel all condition or required, or will leads to proceed with deled volume also

Reply:
eventually if you remove the bracket after or the condition becomes like below

```
if not str(volume.deleted).lower() == 'true' or 
    volume.current_job.get('status', '') in ['', 'finished', 'failed'] or
    volume.vol_id not in [None, ''] or
    volume.name not in [None, '']:
```

So we both are saying same thing ideally :)

Reply:
sorry i mean or is not required, i wrongly committed, and is required

Reply:
Ack